### PR TITLE
[CALCITE] Fixed EXCEPT unparsing bug

### PR DESCRIPTION
**Pull request by AndrewPochapsky**
- Removed overriding of SqlSelect in BigQuery
- Override SelectExpression() in BigQuery

### CODE REVIEW

File: parsing/dialects/intermediate/dialects/bigquery/parserImpls.ftl [hunk 1]
```diff
@@ -15,94 +15,37 @@
...
-    final SqlNodeList windowDecls;
-    final List<SqlNode> hints = new ArrayList<SqlNode>();
-    final Span s;
+    SqlNode e;
 }
 {
-    <SELECT>
-    {
-        s = span();
+    LOOKAHEAD(2)
```
Comment (dasch-1) [inline parsing/dialects/intermediate/dialects/bigquery/parserImpls.ftl:39]: @michaelvdow Isn't 2 the default LOOKAHEAD value?

Comment (AndrewPochapsky) [reply parsing/dialects/intermediate/dialects/bigquery/parserImpls.ftl:39]: I think it used to (still might be) but the code didn't work without this.

Comment (michaelvdow) [reply parsing/dialects/intermediate/dialects/bigquery/parserImpls.ftl:39]: Yeah, it looks like it's still set as 2 in the [build script](https://github.com/googleinterns/calcite/blob/master/parsing/dialects/intermediate/dialects/dialect1/build.gradle.kts#L62)

Comment (AndrewPochapsky) [reply parsing/dialects/intermediate/dialects/bigquery/parserImpls.ftl:39]: Looks like it is actually set to 1 for bigquery: https://github.com/googleinterns/calcite/blob/master/parsing/dialects/intermediate/dialects/bigquery/build.gradle.kts#L62

Comment (dasch-1) [reply parsing/dialects/intermediate/dialects/bigquery/parserImpls.ftl:39]: Aha, good catch. Forgot that it's set to 1 for all others except Dialect1.

Comment (michaelvdow) [reply parsing/dialects/intermediate/dialects/bigquery/parserImpls.ftl:39]: Ah okay, I thought this was for Dialect1 but that makes sense

**Pull request merged**